content/rs/references/crdb-cli-reference.md

```diff
@@ -0,0 +1,284 @@
+
+An Active-Active database requires two or more participating clusters.
+Each instance is responsible for updating the instances residing on other participating clusters with the transactions it receives.
+Write conflicts are resolved using [CRDTs]({{< relref "/rs/administering/active-active.md" >}}).
+
+To programmatically maintain an Active-Active database and its instances, you can use the CRDB-CLI command-line tool.
+With CRDB-CLI you can:
+
+- Create, list, update, flush, or delete an Active-Active database
+- Add or remove an instance of the Active-Active database on a specific cluster
+
+Each command creates a task.
+By default, the command runs immediately and you see the result of the command in the command output.
+If you use the `--no-wait` flag, the command runs in the background so that your application is not delayed by the response.
+You can track the task with [`crdb-cli task status`](#active-active-task-status).
+
+For each CRDB-CLI command, you can use `--help` for additional information about the command.
+
+## Active-Active Database Operations
+
+Active-Active database operations affect the configuration and data of the database.
+
+### Creating an Active-Active Database
+
+The `crdb create` command lets you create a customized Active-Active database and create instances of the database on specified participating clusters. Before you create an Active-Active database you must have:
+
+- At least two participating clusters
+- [Network connectivity]({{< relref "/rs/administering/designing-production/networking/port-configurations.md" >}}) between the participating clusters
+
+When you create an Active-Active database, you must specify:
+
+|Flag and argument| Argument type| Description|
+|---|---|---|
+|`name <CRDB_name>`| string| Name of the Active-Active database|
+|`memory-size <size>`| string| Maximum memory in bytes| kilobytes (kb)| or gigabytes (gb)|
+|`port <port_number>`| integer| TCP port for the Active-Active database on all participating clusters|
+|`instance fqdn=<cluster_fqdn>,username=<username>,password=<password>`| strings| The connection information for the participating clusters. You must specify this flag for each participating cluster.|
+
+The syntax for the command is:
+
+```sh
+crdb-cli crdb create --name <name> /
+--memory-size <maximum_memory> --port <port_number> /
+--instance fqdn=<cluster_fqdn>,username=<username>,password=<password> /
+--instance fqdn=<cluster_fqdn>,username=<username>,password=<password> /
+[--no-wait] /
+[--db-config-json <file_content>] /
+[--db-config-json-file <filename>] /
+[--compression (0-6)] /
+[--causal-consistency true] /
+[--password <password>] /
+[--replication true] /
+[--encryption true] /
+[--sharding false] /
+[-- shards-count <number_of_shards>] /
+[--shard-key-regex <regex_rule>] /
+[--oss-sharding BOOLEAN] /
+[--oss-cluster true]
+[--bigstore true]
+[--bigstore-ram-size <maximum_memory>]
+[--with-module <module_name>]
```

Comment:
What about referencing to a page where we say which modules are supported?

Reply:
We don't have one yet. We're working on it.